composer.json

```diff
@@ -6,26 +6,28 @@
     "license": "Apache-2.0",
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.3|^8.0",
+        "php": "^8.0",
         "ext-dom": "*",
         "ext-openssl": "*",
-        "robrichards/xmlseclibs": "^3.0.4",
-        "simplesamlphp/saml2": "3.2.*",
-        "symfony/dependency-injection": "^4.4|^5.0",
-        "symfony/framework-bundle": "^4.4|^5.0",
-        "symfony/templating": "^4.4|^5.0",
-        "twig/twig": "^2"
+        "robrichards/xmlseclibs": "^3.1.1",
+        "simplesamlphp/saml2": "^4.6.4",
```

Comment:
A significant fix related to the serialization of AttributeValue objects in PHP8+ was applied in saml2 v4.6.5; you may want to bump it.